[2024-12-19T18:24:40.517+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T18:24:40.537+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fintech_etl_pipeline.run_dashboard manual__2024-12-19T18:24:15.930870+00:00 [queued]>
[2024-12-19T18:24:40.548+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fintech_etl_pipeline.run_dashboard manual__2024-12-19T18:24:15.930870+00:00 [queued]>
[2024-12-19T18:24:40.549+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-19T18:24:40.566+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): run_dashboard> on 2024-12-19 18:24:15.930870+00:00
[2024-12-19T18:24:40.578+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=124) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-19T18:24:40.582+0000] {standard_task_runner.py:72} INFO - Started process 144 to run task
[2024-12-19T18:24:40.582+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fintech_etl_pipeline', 'run_dashboard', 'manual__2024-12-19T18:24:15.930870+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/fintech_dag.py', '--cfg-path', '/tmp/tmpfvkjon3m']
[2024-12-19T18:24:40.586+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask run_dashboard
[2024-12-19T18:24:40.651+0000] {task_command.py:467} INFO - Running <TaskInstance: fintech_etl_pipeline.run_dashboard manual__2024-12-19T18:24:15.930870+00:00 [running]> on host 2397727eff2c
[2024-12-19T18:24:40.744+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='fintech_etl_pipeline' AIRFLOW_CTX_TASK_ID='run_dashboard' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T18:24:15.930870+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-19T18:24:15.930870+00:00'
[2024-12-19T18:24:40.746+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T18:24:41.564+0000] {logging_mixin.py:190} INFO - Dash is running on http://0.0.0.0:8050/
[2024-12-19T18:24:41.564+0000] {dash.py:2160} INFO - Dash is running on http://0.0.0.0:8050/

[2024-12-19T18:24:41.587+0000] {logging_mixin.py:190} INFO -  * Serving Flask app 'fintech_dashboard'
[2024-12-19T18:24:41.588+0000] {logging_mixin.py:190} INFO -  * Debug mode: off
[2024-12-19T18:24:41.589+0000] {_internal.py:224} INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8050
 * Running on http://172.19.0.6:8050
[2024-12-19T18:24:41.590+0000] {_internal.py:224} INFO - [33mPress CTRL+C to quit[0m
[2025-02-19T12:11:41.785+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T12:11:41.831+0000] {taskinstance.py:2603} INFO - Dependencies not met for <TaskInstance: fintech_etl_pipeline.run_dashboard manual__2024-12-19T18:24:15.930870+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-02-19T12:11:41.836+0000] {taskinstance.py:2603} INFO - Dependencies not met for <TaskInstance: fintech_etl_pipeline.run_dashboard manual__2024-12-19T18:24:15.930870+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-02-19T12:11:41.841+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
